bundle/edu.gemini.wdba.xmlrpc.server/src/main/java/edu/gemini/wdba/tcc/GMOSSupport.java

```diff
@@ -55,11 +56,17 @@ public String getPositionAngle() {
     }
 
     public String getTccConfigInstrument() {
-        String side = _oe.isNorth() ? "5" : "3";
-        String port = (((InstGmosCommon) _oe.getInstrument()).getIssPort() == IssPort.UP_LOOKING) ? "" : side;
-        String ao   = (_oe.isNorth() && _oe.isAltair()) ? "AO2" : "";
-        String p2   = (_oe.containsTargets(PwfsGuideProbe.pwfs2)) ? "_P2" : "";
-        return ao + "GMOS" + port + p2;
+        final String side = _oe.isNorth() ? "5" : "3";
```

Comment:
This is ridiculous but it corresponds to the ISS port on which the instrument is mounted at the two sites.